src/FsCheck/Runner.fs

```diff
@@ -59,10 +63,11 @@ type Config =
       ///overridden)
       Arbitrary     : list<Type>
       ///A custom test runner, e.g. to integrate with a test framework like xUnit or NUnit. 
-      Runner        : IRunner }
+      Runner        : IRunner
+      ///If set, inputs for property generation and property evaluation will be runned in parallel. 
+      ParallelRunConfig : ParallelRunConfig option}
```

Comment:
Do you have any other options in mind? If not would prefer to just put the one field into the Config record.